pkg/ovsdb/transact.go

```diff
@@ -45,7 +45,7 @@ const (
 )
 
 func isEqualSet(expected, actual interface{}) bool {
-	return reflect.DeepEqual(expected, actual)
+	return len(expected.(libovsdb.OvsSet).GoSet) == len(actual.(libovsdb.OvsSet).GoSet) && isIncludesSet(expected, actual)
 }
 
 func isIncludesSet(expected, actual interface{}) bool {
```

Comment:
The current implementation compares "==" and "include"  as the same. It was one of our long time ago misconceptions. I think we can change this and for "include" Maps and Sets use "isIncludesSet" , "isIncludemap" instead of "isEqual.."

Reply:
you are right, `==` and `include` are not the same. I updated the code.
also `!=` and `exclude` are not interchangeable anymore  on `OvsMap` and `OvsSet` and I've update it too.